src/Core/src/PublicAPI/net-android/PublicAPI.Shipped.txt

```diff
@@ -3272,7 +3272,6 @@ virtual Microsoft.Maui.PropertyMapper.ClearKeyCache() -> void
 virtual Microsoft.Maui.PropertyMapper.GetKeys() -> System.Collections.Generic.IEnumerable<string!>!
 virtual Microsoft.Maui.PropertyMapper.GetProperty(string! key) -> System.Action<Microsoft.Maui.IElementHandler!, Microsoft.Maui.IElement!>?
 virtual Microsoft.Maui.PropertyMapper.SetPropertyCore(string! key, System.Action<Microsoft.Maui.IElementHandler!, Microsoft.Maui.IElement!>! action) -> void
-virtual Microsoft.Maui.PropertyMapper.UpdateKeys.get -> System.Collections.Generic.IReadOnlyCollection<string!>!
```

Comment:
this needs to stay

Reply:
May I ask why?

Reply:
@albyrock87 it confuses me every time too. We already shipped this API, that didn't change. In order to remove it I should not remove this line but only add a same line with *REMOVED* in front in the Unshipped.txt. We don't mean that we need to keep this actual API, just the line in this file

Reply:
Right, ... I need some rest!

Reply:
No worries my friend, I totally understand. I was talking to Rui about this directly so I had a bit more context ;)